### DISCUSSION TITLE
Duplicate Themes

### DISCUSSION THREAD

**Discussion by RookiexCookie** [prasanthrangan/hyprdots#1450] [Q&A] [answered]
https://github.com/prasanthrangan/hyprdots/assets/136296017/28eea43c-479b-48a6-b283-3fbe109b7d63

i rebuilded the themes and reinstalled Hyde bunch of times and removed ~/.cache/hyprdots .config/swww and .config/hyprdots but still got 2 of these

**Comment by kRHYME7** [comment, marked as answer]
Fix the vid.  

You can remove themes by just removing the folder name. 
![image](https://github.com/prasanthrangan/hyprdots/assets/53417443/ee4a5ed4-9ea4-43fb-a01f-26add151ae36)

**Reply by RookiexCookie** [reply]
Thanks 
btw what you mean fix the vid?

**Reply by kRHYME7** [reply]
MyB, just firefox video rendering issues
